[2024-12-16T20:00:02.981+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sick_leave_etl_dag.load_data scheduled__2024-12-16T13:00:00+00:00 [queued]>
[2024-12-16T20:00:02.987+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sick_leave_etl_dag.load_data scheduled__2024-12-16T13:00:00+00:00 [queued]>
[2024-12-16T20:00:02.988+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2024-12-16T20:00:03.000+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): load_data> on 2024-12-16 13:00:00+00:00
[2024-12-16T20:00:03.004+0000] {standard_task_runner.py:57} INFO - Started process 6705 to run task
[2024-12-16T20:00:03.006+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'sick_leave_etl_dag', 'load_data', 'scheduled__2024-12-16T13:00:00+00:00', '--job-id', '71', '--raw', '--subdir', 'DAGS_FOLDER/sick_leave_dag.py', '--cfg-path', '/tmp/tmplxhozfqb']
[2024-12-16T20:00:03.008+0000] {standard_task_runner.py:85} INFO - Job 71: Subtask load_data
[2024-12-16T20:00:03.023+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-12-16T20:00:03.055+0000] {task_command.py:415} INFO - Running <TaskInstance: sick_leave_etl_dag.load_data scheduled__2024-12-16T13:00:00+00:00 [running]> on host cfbf61ae1f16
[2024-12-16T20:00:03.129+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sick_leave_etl_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-16T13:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-16T13:00:00+00:00'
[2024-12-16T20:00:03.143+0000] {base.py:73} INFO - Using connection ID 'admin' for task execution.
[2024-12-16T20:00:03.174+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-12-16T20:00:03.184+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=sick_leave_etl_dag, task_id=load_data, execution_date=20241216T130000, start_date=20241216T200002, end_date=20241216T200003
[2024-12-16T20:00:03.218+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-12-16T20:00:03.234+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
